transport/grpc/server.go

```diff
@@ -68,8 +68,19 @@ func ServerAfter(after ...ServerResponseFunc) ServerOption {
 
 // ServerErrorLogger is used to log non-terminal errors. By default, no errors
 // are logged.
+// Deprecated: Use ServerErrorHandler instead.
 func ServerErrorLogger(logger log.Logger) ServerOption {
-	return func(s *Server) { s.logger = logger }
+	return func(s *Server) {
+		if s.errorHandler == nil {
+			s.errorHandler = log.NewErrorHandler(logger)
+		}
+	}
+}
+
+// ServerErrorHandler is used to handle non-terminal errors. By default, no errors
+// are handled.
```

Comment:
I think it would be a little bit less ambiguous to say e.g. "By default, non-terminal errors are ignored."